packages/fxa-auth-server/scripts/stripe-products-and-plans-to-firestore-documents/stripe-products-and-plans-converter.ts

```diff
@@ -389,10 +475,22 @@ export class StripeProductsAndPlansConverter {
         if (isDryRun) {
           // We'll log the full productConfig with `locales` updated further down
         } else {
-          productConfigId = await this.paymentConfigManager.storeProductConfig(
-            productConfig,
-            productConfigId
-          );
+          if (target === OutputTarget.Firestore) {
+            productConfigId =
+              await this.paymentConfigManager.storeProductConfig(
+                productConfig,
+                productConfigId
+              );
+          } else if (target === OutputTarget.Local) {
+            // Create the folder here.
+            await createDir(currentDirectory);
```

Comment:
Should we move the `await createDir()` into the `writeToFileProductConfig()` fn so we always confirm the folder exists before attempting to write the file?

I think packages/fxa-auth-server/scripts/stripe-products-and-plans-to-firestore-documents/stripe-products-and-plans-converter.ts#L556 below calls `await this.writeToFileProductConfig()` but without the `await createDir()`; although maybe that's because it's guaranteed to be created here.

Reply:
I'd like to keep it separate for now. At the moment the assumption is that the Dir will be created at the top of `convert`, and the `writeToFile*` functions assume the dir exists.